examples/flutter_gallery/lib/demo/rally/finance.dart

```diff
@@ -0,0 +1,332 @@
+// Copyright 2019-present the Flutter authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+import 'package:flutter_gallery/demo/rally/charts/pie_chart.dart';
+import 'package:flutter_gallery/demo/rally/charts/line_chart.dart';
+import 'package:flutter_gallery/demo/rally/charts/vertical_fraction_bar.dart';
+import 'package:flutter_gallery/demo/rally/colors.dart';
+import 'package:flutter_gallery/demo/rally/data.dart';
+import 'package:flutter_gallery/demo/rally/formatters.dart';
+
+class FinancialEntityView extends StatelessWidget {
+  const FinancialEntityView({
+    this.heroLabel,
+    this.heroAmount,
+    this.wholeAmount,
+    this.segments,
+    this.financialEntityCards,
+  }) : assert(segments.length == financialEntityCards.length);
+
+  /// The amounts to assign each item.
+  ///
+  /// This list must have the same length as [colors].
```

Comment:
colors?

Reply:
I will delete this comment as I think this comment got out of date, because there is not any limit for the segments length